src/site/content/en/fast/font-best-practices/index.md

```diff
@@ -135,41 +84,34 @@ stylesheet to download.
 
     body {
         font-family: "Open Sans";
-
     }
+
+    ...etc.
+
   </style>
 </head>
-
 ```
 
 {% Aside %}
-    Inlining the font files themselves is not recommended. Inlining large
-    resources like fonts is likely to delay the delivery of the main document,
-    and with it, the discovery of other resources.
-
+    Note that if only some of the CSS is inlined, then the browser will still need to wait for all the CSS to be loaded, before it can discover if fonts are needed.<br>
+    <br>
+    Also note that inlining the font files themselves is not recommended. Inlining large resources like fonts is likely to delay the delivery of the main document, and with it, the discovery of other resources.
 {% endAside %}
 
 
-#### Preconnect to critical third-party origins
+Inlining critical CSS can be a more advanced technique that not all sites will be able to achieve. The performance benefits are clear, but it requires additional processes and build tools to ensure the necessarily CSS—and ideally only the critical CSS—is inlined correctly and that any additional CSS is delivered in a non-render blocking fashion.
 
-If your site loads fonts from a third-party site, it is highly recommended that
-you use the
-[`preconnect`](https://developer.mozilla.org/docs/Web/HTML/Link_types/preconnect)
-resource hint to establish early connection(s) with the third-party origin.
-Resource hints should be placed in the `<head>` of the document. The resource
-hint below sets up a connection for loading the font stylesheet.
+### Preconnect to critical third-party origins
 
+If your site loads fonts from a third-party site, it is highly recommended that you use the [`preconnect`](https://developer.mozilla.org/docs/Web/HTML/Link_types/preconnect) resource hint to establish early connection(s) with the third-party origin. Resource hints should be placed in the `<head>` of the document. The resource hint below sets up a connection for loading the font stylesheet.
 
 ```html
 <head>
   <link rel="preconnect" href="https://fonts.com">
```

Comment:
What if we use the Google Fonts snippet here, so we have a more concrete example?